prompt.go

```diff
@@ -53,10 +119,104 @@ func (i *InitActionCmd) setDriver() error {
 		},
 	}
 	_, driver, err := prompt.Run()
-	i.Driver = driver
+	i.driver = driver
 	return err
 }
 
+func (i *InitActionCmd) selectAtlasRepo(ctx context.Context, cloud cloudapi.API) (*cloudapi.Repo, error) {
+	var choose int
+	repos, err := cloud.Repos(ctx)
+	switch {
+	case err != nil:
+		return nil, err
+	case len(repos) == 0:
+		return nil, errors.New("no repositories found")
+	case i.To != "": // Search by URL
+		choose = slices.IndexFunc(repos, func(r cloudapi.Repo) bool {
+			return r.URL == i.To
+		})
+		if choose == -1 {
+			return nil, errors.New("no repository with given URL found")
+		}
+	case i.DirName != "": // Search by directory name
+		choose = slices.IndexFunc(repos, func(r cloudapi.Repo) bool {
+			return r.Slug == i.DirName
+		})
+		if choose == -1 {
+			return nil, errors.New("no repository with given name found")
+		}
+	case len(repos) == 1:
+		choose = 0
+	default:
+		prompt := promptui.Select{
+			Label: "Select an Atlas Cloud Repository",
+			Items: repos,
+			Stdin: i.stdin,
+			Templates: &promptui.SelectTemplates{
+				Active:   `▸ {{ .Title }}`,
+				Inactive: `  {{ .Title }}`,
+				Selected: fmt.Sprintf(`{{ "%s" | green }} {{ "Selected repository:" | faint }} {{ .Title }}`, promptui.IconGood),
+			},
+		}
+		choose, _, err = prompt.Run()
+		if err != nil {
+			return nil, err
+		}
+	}
+	return &repos[choose], nil
+}
+
+func (i *InitActionCmd) setDesiredState() error {
+	if i.To != "" {
+		return nil
+	}
+	prompt := promptui.Prompt{
+		Label: "Enter a URL of the desired schema state",
+		Stdin: i.stdin,
+		Validate: func(input string) error {
+			if len(i.To) == 0 && strings.TrimSpace(input) == "" {
+				return errors.New("at least one URL is required for desired schema state")
+			}
+			return nil
+		},
+		Templates: &promptui.PromptTemplates{
+			Success: fmt.Sprintf(`{{ "%s" | green }} {{ "Desired schema state URL: " | faint }}`, promptui.IconGood),
+		},
+	}
+	f, err := prompt.Run()
+	if err != nil {
+		return err
+	}
+	i.To = f
+	return nil
+}
+
+func (i *InitActionCmd) setCurrentState() error {
+	if i.From != "" {
+		return nil
+	}
+	prompt := promptui.Prompt{
+		Label: "Enter a URL of the current schema state",
+		Stdin: i.stdin,
+		Validate: func(input string) error {
+			if len(i.From) == 0 && strings.TrimSpace(input) == "" {
+				return errors.New("at least one URL is required for current schema state")
+			}
+			return nil
+		},
+		Templates: &promptui.PromptTemplates{
+			Valid:   fmt.Sprintf(`{{ "%s" | green }} {{ . | faint }} {{": " | faint }}`, promptui.IconGood),
+			Success: fmt.Sprintf(`{{ "%s" | green }} {{ "Current schema state URL: " | faint }}`, promptui.IconGood),
```

Comment:
```suggestion
			Success: fmt.Sprintf(`{{ "%s" | green }} {{ "Current schema state URL:" | faint }}`, promptui.IconGood),
```
![image](https://github.com/user-attachments/assets/dee48c9d-b097-4893-8a8c-0bce781d1002)